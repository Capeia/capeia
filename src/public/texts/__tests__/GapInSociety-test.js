// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<GapInSociety>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/GapInSociety')
    const Component = require('public/texts/GapInSociety').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
