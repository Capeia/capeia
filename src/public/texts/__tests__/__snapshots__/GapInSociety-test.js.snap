// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GapInSociety> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="contentContainer  col-lg-6 col-lg-offset-3 col-md-8"
    >
      <div
        className={undefined}
      >
        <div
          className=""
        >
          <h1>
            A Gap in Society
          </h1>
          <p>
            As a Vienna-based science portal we would like to remember the vibrant Jewish community that has contributed so strongly to Austria’s cultural heritage.
          </p>
          <p>
            Jewish scientists, physicians, artists, and intellectuals were the main protagonists of a landmark epoch in Western civilisation which has become known as the 
            <em>
              Viennese Fin-de-siècle
            </em>
            . During this period the arts and sciences were fostered by a dedicated and open-minded citizenry that was largely Jewish or of Jewish origin.
          </p>
          <p>
            Alas, this exceptionally fruitful epoch was not to last. Well before the events that are now referred to as the Holocaust or the Shoah, which brought about the physical and intellectual eradication of Jewish life in Austria, antisemitism was already on the rise. Jewish scientists were discriminated against by ignoring their achievements and by refusing them positions at, or dismissing them from, universities.
          </p>
          <p>
            Sadly, following the end of WWII and the official “de-nazification” of Austrian society, no serious attempts at compensation were made and no visible efforts undertaken to restore the once so vital Jewish community in Austria.
          </p>
          <p>
            The inspiring intellectual climate is painfully missed to the present day, both within academia and elsewhere.
          </p>
        </div>
      </div>
    </div>
    <div
      className="sidebarContainer col-lg-3 col-md-4"
    >
      <aside
        className="sidebar"
      />
    </div>
  </div>
</div>
`;
